lua/dylib.lua

```diff
@@ -30,6 +71,81 @@ int add_fib_entry(const char *prefix, const char *gt_ip, const char *gw_ip,
 	enum gk_fib_action action, struct gk_config *gk_conf);
 int del_fib_entry(const char *ip_prefix, struct gk_config *gk_conf);
 
+struct ipaddr *get_fib_dump_entry_ip(struct gk_fib_dump_entry *entry);
+struct ipaddr *get_fib_dump_entry_grantor_ip(struct gk_fib_dump_entry *entry);
+struct ipaddr *get_fib_ether_dump_entry_nexthop(
+	struct gk_fib_ether_dump_entry *entry);
+struct ether_addr *get_fib_ether_dump_entry_mac(
+	struct gk_fib_ether_dump_entry *entry);
+int convert_ip_to_str(struct ipaddr *ip_addr, char *res, int n);
+void ether_format_addr(char *buf, uint16_t size,
+	const struct ether_addr *eth_addr);
+
 ]]
 
 c = ffi.C
+
+function print_fib_dump_entry(fib_dump_entry, arg)
```

Comment:
1. Add the following comment before `print_fib_dump_entry()`:
```
The following is an example function that can be used as the callback function of list_fib4_entries() and list_fib6_entries().

Parameter fib_dumb_entry is going to be released after print_fib_dump_entry() returns, so don't keep references to fib_dumb_entry or any of the data reachable through its fields.
```
Please adjust lines as needed.

2. Drop parameter `arg` it doesn't do anything in Lua. If a Lua script needs a fixed parameter to be added to a handler, it just uses a closure.